### ISSUE TITLE
Items before first sticky header causes ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by msa1422** [Hospes/headed-lazy-grid#2]
When a few items are added before the first sticky header or if 2 sticky headers are very close to each other with only 2-3 rows between them, reverse direction scroll causes a crash with ArrayIndexOutOfBoundsException

Reproduce the issue by adding this code before the first stickyHeader in MainActivity
```
LazyVerticalGrid(...) {

    // Items before header
    items(items = arrayListOf(testItems[0], testItems[1])) {
        Item(
            item = it,
            modifier = Modifier.fillMaxWidth(),
        )
     }

    // First Header
    stickyHeader {
        ...
    }

    // Rest of the code
    ...

}
```

Alternatively, you can check this [fork](https://github.com/msa1422/headed-lazy-grid)

**Comment by Hospes** [comment]
Thanks for the catch ! I'll try to check it later this week. Actually hope google will provide native solution for us in couple of weeks :)

**Comment by Hospes** [comment]
hi there, I'm sorry that it took so long for me to fix that issue.
I've released new version of LazyGrid [0.7.1](https://github.com/Hospes/headed-lazy-grid/releases/tag/0.7.1) (updated source code from 1.4.0-alpha03)
Hope that gonna help you!
